src/main/java/pt/lighthouselabs/obd/commands/ObdCommand.java

```diff
@@ -28,6 +28,7 @@
   protected String cmd = null;
   protected boolean useImperialUnits = false;
   protected String rawData = null;
+  protected long responseTimeDelay = 200;//automatic sampling rate detection coming
```

Comment:
I plan to measure this for each command and set it accordingly, so neither final nor static apply..